common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +456,199 @@ def test_teams_membership(self):
+        And When I click create team link
+        Then I should see the create team page.
+        And I should see the create team header
+        And I should also see the help messages for fields.
+        """
+        self.verify_and_navigate_to_create_team_page()
+        self.assertEqual(
+            self.create_team_page.message_for_field('name'),
+            'A name that identifies your team (maximum 255 characters).'
+        )
+        self.assertEqual(
+            self.create_team_page.message_for_textarea_field('description'),
+            'A short description of the team to help other learners understand '
+            'the goals or direction of the team (maximum 300 characters).'
+        )
+        self.assertEqual(
+            self.create_team_page.message_for_field('country'),
+            'The country that team members primarily identify with.'
+        )
+        self.assertEqual(
+            self.create_team_page.message_for_field('language'),
+            'The language that team members primarily use to communicate with each other.'
+        )
+
+    def test_user_can_see_error_message_for_missing_data(self):
+        """
+        Scenario: The user should be able to see error message in case of missing required field.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Create Team page for that topic
+        Then I should see the Create Team header and form
+        And When I click create team button without filling required fields
+        Then I should see the error message and highlighted fields.
+        """
+        self.verify_and_navigate_to_create_team_page()
+        self.create_team_page.submit_form()
+
+        self.assertEqual(
+            self.create_team_page.validation_message_text,
+            'Check the highlighted fields below and try again.'
+        )
+        self.assertTrue(self.create_team_page.error_for_field(field_id='name'))
+        self.assertTrue(self.create_team_page.error_for_field(field_id='description'))
+
+    def test_user_can_see_error_message_for_incorrect_data(self):
+        """
+        Scenario: The user should be able to see error message in case of increasing length for required fields.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Create Team page for that topic
+        Then I should see the Create Team header and form
+        When I add text > than 255 characters for name field
+        And I click Create button
+        Then I should see the error message for exceeding length.
+        """
+        self.verify_and_navigate_to_create_team_page()
+        self.create_team_page.submit_form()
+
+        # Fill the name field with >255 characters to see validation message.
+        self.create_team_page.value_for_text_field(
+            field_id='name',
+            value='EdX is a massive open online course (MOOC) provider and online learning platform. '
```

Comment:
@muhammad-ammar did you see the above comment? I'd rather not include references to other MOOC companies in our test code.